### PR TITLE
[RESTEASY-3062] rewrite section 3.1.2

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: docbook/reference/en/en-US/modules/Installation_Configuration.xml [hunk 1]
```diff
@@ -257,20 +257,35 @@
 	    <section id="upgrading-wildfly">
 	        <title>Upgrading RESTEasy within WildFly</title>
 	        <para>
-                RESTEasy is bundled with WildFly. However you may wish to upgrade to the latest version. With
-                <ulink url="https://docs.wildfly.org/24/Galleon_Guide.html">Galleon</ulink> this makes upgrading RESTEasy
-                in WildFly quite easy.
+               For developers working in a non-production environment there are several
+               options for installing an alternate version of RESTEasy in a local copy
+               of Wildfly.
```
Comment (jamezp) [inline docbook/reference/en/en-US/modules/Installation_Configuration.xml:262]: Minor nit, this should be "WildFly" with a capital F.

File: docbook/reference/en/en-US/modules/Installation_Configuration.xml [hunk 2]
```diff
@@ -257,20 +257,35 @@
...
-                <programlisting><![CDATA[$ galleon.sh install wildfly:current]]></programlisting>
-
-                To install the RESTEasy upgrade on top of that you simply need to use the tool again with the Maven GAV:
-                <programlisting><![CDATA[$ galleon.sh install org.jboss.resteasy:galleon-feature-pack:6.0.0-SNAPSHOT]]></programlisting>
+                The following options assume there is a local build of the RESTEasy project,
+                version 6.x.x or higher and a local installation of Wildfly version 25.x.x or higher.
             </para>
+           <para>
+               Option 1, if Wildfly project source code was installed locally from a zip file or there
+               is a local build of the Wildfly project, then Wildfly will need to be rebuilt
```
Comment (jamezp) [inline docbook/reference/en/en-US/modules/Installation_Configuration.xml:270]: Same here with WildFly.

**Review by jamezp — COMMENTED** [review]
Only change here would be making all the references of "Wildfly" to "WildFly".

**Comment by rsearls** [comment]
Obsolete

**Pull request closed without merging**